src/ui/server/helpers/mappings/map-yes-no-field.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * mapYesNoField
+ * maps true or false fields to Yes/No
+ * @param {Boolean} answer
+ * @returns {String} Yes/No or original provided answer
+ */
+const mapYesNoField = (answer?: boolean) => {
```

Comment:
Do we need the param type to be optional? If so we might need to add a check in the function to make sure `answer` exists